CMSgov/design-system
# [NO-TICKET] Clean up examples

## Summary

- Updated the `create-react-app-typescript` example to match the latest template from the CRA project and to use the latest version of the design system
- Removed the plain `create-react-app` example because there is too much overlap with the `create-react-app-typescript` example to justify maintaining both
- Removed the `webpack-demo` example because it's a lot of code that gets out of date quickly, and the other examples cover the setup well enough. We're not a tutorial on how to set up Webpack.
- Cleaned up the `html-css` example and have it us the CDN
- **Question to @forrestbaer:** What if the `cdn` example referenced the assets straight from the CDN instead of with a local build and copy? (See my updates to the `html-css` example)

## How to test

`cd` into each of the examples and make sure you can use the README to get them each to run

## Checklist




### If this is a change to documentation:
